packages/block-library/src/image/interactivity.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Internal dependencies
+ */
+import { store } from '../utils/interactivity';
+
+const raf = window.requestAnimationFrame;
+// Until useSignalEffects is fixed: https://github.com/preactjs/signals/issues/228
+const tick = () => new Promise( ( r ) => raf( () => raf( r ) ) );
```

Comment:
Thanks for the feedback!

Visibility is is in the animation to ensure the overlay doesn't disappear before the opacity reaches 0 in the fade out animation.

That being said, I realized that we actually don't need that during the fade in animation, and since that's the only time we have the focus problem, we can remove the visibility attribute in that case.

I've gone ahead and made that change, as well as removed the `tick()` 🙌